source/blog/2018-09-28-the-ember-times-issue-66.md

```diff
@@ -52,6 +52,18 @@ There has been [some documentation missing](https://github.com/emberjs/ember.js/
 
 Going forward the plan is to provide better testing for cases like this so you always have access to the documentation you need.
 
+---
+
+## Project Structure for Single Page Apps
```

Comment:
Do you mind adding the link to your blog post in the title, too? You can also take a look at the other sections for reference if you like